src/mir/dataflow.md

```diff
@@ -154,8 +154,24 @@ for (bb, block) in body.basic_blocks().iter_enumerated() {
 }
 ```
 
+### Graphviz Diagrams
+
+When the results of a dataflow analysis are not what you expect, it often helps
+to visualize them. This can be done with the `-Zdump-mir` flags described in
+[Debugging MIR]. Start with `-Zdump-mir=F -Zdump-mir-dataflow`, where `F` is
+either "all" or the name of the MIR body you are interested in.
+
+These `.dot` files will be saved in your `mir_dump` directory and will have the
+`NAME` of the analysis (e.g. `maybe_inits`) as part of their filename. Each
```

Comment:
Is `NAME` a jargon term here? It seems to appear all of a sudden.

Reply:
It's an associated constant for the `AnalysisDomain` trait. I can link to it maybe?

Reply:
That would be helpful, thanks :)